pkgs/development/python-modules/pytaglib/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, taglib
+, cython
+}:
+
+buildPythonPackage rec {
+  pname   = "pytaglib";
+  version = "1.4.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "44ab26dc4b33962b8db0bb8856e7b166539c0c555bc933a6bbbc96f4ec51c7a2";
+  };
+
+  buildInputs = [ taglib cython ];
+
+  meta = {
+    homepage = https://github.com/supermihi/pytaglib;
+    description = "Python 2.x/3.x bindings for the Taglib audio metadata library";
+    license = lib.licenses.gpl3;
```

Comment:
Do you want to maintain this package? https://nixos.wiki/wiki/Nixpkgs#Becoming_a_Nixpkgs_maintainer

Reply:
Well I could, but I'm either a Python or Nix expert :)

Reply:
The point is, if you can test if pytaglib works or not.

Reply:
OK, I can be a maintainer then. I’ll add myself in this PR later today.